src/adapter/src/coord/sequencer/inner/explain_timestamp.rs

```diff
@@ -261,7 +261,7 @@ impl Coordinator {
                         .unwrap_or_else(|| id.to_string());
                     sources.push(TimestampSource {
                         name: format!("{name} ({id}, compute)"),
-                        read_frontier: state.read_capability().elements().to_vec(),
+                        read_frontier: state.read_frontier().to_vec(),
```

Comment:
I'm pretty sure returning the implied capability here was wrong. In EXPLAIN TIMESTAMP you either want to see the collection's actual read frontier, or the frontier at which you can still make a query, both of with are the `read_frontier()`, not the implied capability.